Makefile

```diff
@@ -51,6 +51,10 @@ compile: mod ## Compile for the local architecture ⚙
 	-X 'main.Name=$(target)'" \
 	-o bin/$(target) .
 
+.PHONY: tools
+tools: ## Install tools 🛠
+	go install github.com/goreleaser/goreleaser@latest
```

Comment:
A few notes.
1. If we keep this, perhaps we should pin a specific version
2. We could consider changing this to `go run` semantics, which would always run a specific version (rather than whatever a given user has installed) and also doesn't require `GOBIN` being in `PATH`

Reply:
sure, I tend to always use latest for tools as using the latest tools surfaces deprecations early and minimizes tech debt.

I think the go run semantics would be fine, we can then remove this and just instruct people to run it. or make a `make goreleaser` which then just runs `go run  github.com/goreleaser/goreleaser@v1.14.0` so people can do `make goreleaser --snapshot --rm-dist` for local building

Reply:
Ok, I have made the changes in the latest commit. 

Pinned to `1.14` so that any patch releases are pulled automatically, like `1.14.1` if it happens. Did the same for the github action workflow.

Updated releasing.md to include info about the adjusted make commend usage for local building if needed.

Reply:
At some point, using the docker builder pattern instead which could define all the tools?  Is that a pattern others use or is it something that seems more niche (I thought it was pretty common but I'm finding less of it than I thought).

Reply:
> At some point, using the docker builder pattern instead which could define all the tools?  Is that a pattern others use or is it something that seems more niche (I thought it was pretty common but I'm finding less of it than I thought).

It's common internally at companies, but the volume mounting aspect causes problems and usability issues 

Teams tend to just version manage things like with `asdf` or something

I rarely see it in open source projects. Maybe nix might be a better setup?